spec/models/concerns/digital_object_concerns/digital_object_data/setters/rights_spec.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe DigitalObjectConcerns::DigitalObjectData::Setters::Rights do
+  let(:digital_object_with_sample_data) { FactoryBot.build(:digital_object_test_subclass, :with_sample_data) }
+
+  context '#clean_rights!' do
+    let(:rights) do
+      {
+        'descriptiveMetadata' => [
+          {
+            'typeOfContent' => '  compilation ',
+            'countryOfOrigin' => nil,
+            'filmDistributedToPublic' => nil,
+            "filmDistributedCommercially" => nil
+          }
+        ],
+        'copyrightStatus' => [
+          {
+            'copyrightNote' => '   ',
+            'copyrightRegistered' => true,
+            'copyrightRenewed' => true,
```

Comment:
Do you mind if we change this `true` to `false` just to prove that `false` values aren't removed by the cleaning?